test/OpenTelemetry.Tests/Metrics/MetricOverflowAttributeTestsBase.cs

```diff
@@ -104,7 +104,7 @@ public void TestEmitOverflowAttributeConfigWithOtherConfigProvider(string value,
     }
 
     [Theory]
-    [InlineData(1, false)]
+    [InlineData(1, true)]
```

Comment:
The right way to update this test would be to rename it to `EmitOverflowAttributeIsNotDependentOnMaxMetricPoints` and remove the `isEmitOverflowAttributeKeySet` parameter from the test method as we always want it to be `true`.